defaults/main.yml

```diff
@@ -1,7 +1,10 @@
+mysql_hardening_enabled: yes
 # general configuration
 mysql_hardening_user: 'mysql'
+mysql_hardening_group: 'mysql'
 mysql_datadir: '/var/lib/mysql'
 mysql_hardening_hardening_conf: '/etc/mysql/conf.d/hardening.cnf'
+mysql_root_password: "{{ lookup('env','mysql_root_password') }}"
```

Comment:
We could use a separate variables-file that only contains the password and encrypt this file with vault. Or something along these lines.